docs/source/tutorial.rst

```diff
@@ -313,13 +313,143 @@ set in both ``experiment_definition`` and extensions, the one set in the extensi
 Executing the experiment
 ------------------------
 
+Following either of the approaches listed below, the experiment definition set up could be executed:
+
+One step execution
+##################
 The experiment definition set up can now be passed to the ``execute()`` function::
 
-   exp = workspace.execute(experiment_definition).wait()
+   experiment = workspace.execute(experiment_definition).wait()
+
+This approach couples both the creation and execution of the experiment in one step. Executing this multiple
+time would create a new experiment object, hence is not advisable if the user wishes to re-execute the same 
+experiment multiple times.
```

Comment:
ln 325 time should be plural...
Perhaps:
Subsequent executions will create a new experiment object and as such ill advised for re-execution of the same experiment.